sage-ios/SAGE/SAGE/ProfileView.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  ProfileView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 11/21/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class ProfileView: UIView {
+    
+    let profileContent = UIView()
+    let header = UIView()
+    let profileUserImg = UIImageView()
+    let profileUserImgBorder = UIImageView()
+    let profileEditButton = UILabel()
+    let userName = UILabel()
+    let userVolunteerLevel = UILabel()
+    let userSchool = UILabel()
+    let topBorder = UIView()
+    let bottomBorder = UIView()
+    let centerBorder = UIView()
+    let userStatusContainer = UIView()
+    let userStatusLabel = UILabel()
+    let userCommitmentContainer = UIView()
+    let userCommitmentLabel = UILabel()
+    let logoutButton = UITableViewCell()
```

Comment:
wait do you even need the all check ins or logout button in this view since its in the tableview now?